### ISSUE TITLE
Only first video outputs audio through Unity on Android with AVPro-v3

### ISSUE THREAD

**Issue by Last8Exile** [RenderHeads/UnityPlugin-AVProVideo#1836]
**Describe the issue**
We using AudioMode: Unity.
First video play audio through Unity.
All further videos play audio bypassing Unity.

We use additional sound processing on Unity side (spatialization, 5.1 to stereo mixing, gloabal volume control through mixer groups)
and rely on AudioMode: Unity working correctly.
Workaround would be helpfil.

**Your Setup (please complete the following information):**
 - Unity version: 2022.3.24f1
 - AVPro Video version: 3.0.0f1-ultra
 - Operating system version: Android 14, Android 12
 - Device model: Oculus Quest 2 and 3, RedMagic 9, Google Pixel 6a
 - Video specs (resolution, frame-rate, codec, file size): container: MP4, video: AVC, audio: AAC (seems unrelated)

**To Reproduce**

0. Open attached project (link in email)
1. Build and run application (Development Build) on Android.
2. Connect Unity Profiler with Audio module to application running on android.
3. Enter path to any local video with audio ( /storage/emulated/0/.../someVideo.mp4 ) in InputField at the Top.
4. Tap Open button at Top Right
5. Tap Play button at Bot Left (or at screen Center)
6. Observe audio is played through Unity
6.1. Green spectrum visualisation on Bot Right is not flat
6.2. VU Level of Global mixer group is above -100 db
7. Tap Open button again
8. Tap Play button again
9. Observe audio is played bypasing Unity
9.1. Green spectrum visualisation on Bot Right is flat
9.2. VU Level of Global mixer group is below -100 db (it needs several seconds to drop)

**Comment by Last8Exile** [comment]
Workaround: creating new player each time opening new video fixes the issue.
Simple copy and destroy approach:
```
MediaPlayer _player;
...
var oldPlayer = _player;
_player = Instantiate(oldPlayer);
SetReferencesAndResubscribeToEventsInExternalComponents(_player);
Destroy(oldPlayer.gameObject);
```

**Comment by Chris-RH** [comment]
Thanks, we have received your email

**Comment by Last8Exile** [comment]
Workaround with recreating players somehow randomly freezes main thread completly.
 
It freezes somewhere near executing MediaPlayer.OnDestroy. If I don't destoy previous MediaPlayer instances and only disable them - there is no freeze. I can't reproduce it on Demo project and ran out of time to investigate.

**Comment by Ste-RH** [comment]
Can you point me to the scene I should be opening in the project you sent over please @Last8Exile ?

**Comment by Ste-RH** [comment]
A fix for this is offered in v3.0.2 (which we are hoping to release today)

**Comment by Chris-RH** [comment]
AVPro Video version 3.0.2 has been released.  Please let us know if it has not fixed your issue.
